packages/sources/mycryptoapi/src/endpoint/gasprice.ts

```diff
@@ -0,0 +1,35 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+
+export const supportedEndpoints = ['gasprice']
+
+interface ResponseSchema  {
```

Comment:
Let's export this as our core framework might use it in the future.

Reply:
done!